spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -33,27 +34,35 @@ However, any application registered with a Maven, HTTP, or File resource for the
 
 === Kubernetes Compatibility
 
-The Spring Cloud Data Flow implementation for Kubernetes uses the link:https://github.com/spring-cloud/spring-cloud-deployer-kubernetes[Spring Cloud Deployer Kubernetes]
-library for orchestration. Before you begin setting up a Kubermetes cluster, refer to the link:https://github.com/spring-cloud/spring-cloud-deployer-kubernetes#kubernetes-compatibility[compatibility matrix]
-to learn more about deployer and server compatibility against Kubernetes release versions.
+The Spring Cloud Data Flow implementation for Kubernetes uses the link:https://github.com/spring-cloud/spring-cloud-deployer-kubernetes[Spring Cloud Deployer Kubernetes] library for orchestration.
+Before you begin setting up a Kubermetes cluster, refer to the link:https://github.com/spring-cloud/spring-cloud-deployer-kubernetes#kubernetes-compatibility[compatibility matrix] to learn more about deployer and server compatibility against Kubernetes release versions.
 
 === Create a Kubernetes Cluster
 
 The Kubernetes https://kubernetes.io/docs/setup/pick-right-solution/[Picking the Right Solution] guide lets you choose among many options, so you can pick the one that you are most comfortable using.
 
-All our testing is done with the https://cloud.google.com/kubernetes-engine/[Google Kubernetes Engine] that is part of the Google Cloud Platform. That is also the target platform for this section. We have also successfully deployed with https://kubernetes.io/docs/getting-started-guides/minikube/[Minikube]. We note where you need to adjust for deploying on Minikube.
+All our testing is done with the https://cloud.google.com/kubernetes-engine/[Google Kubernetes Engine] that is part of the Google Cloud Platform.
```

Comment:
@chrisjs: Since now we have ATs running on PKS, we could relax this and include PKS as another target.

Reply:
sure, works for me

Reply:
@sabbyanandan , @chrisjs who this changes the documentation?

Reply:
Feel free to revise it while you're at this and the PR.